data/language/en-GB.txt

```diff
@@ -3456,6 +3456,7 @@ STR_3443    :Page 4
 STR_3444    :Page 5
 STR_3445    :Set Patrol Area
 STR_3446    :Cancel Patrol Area
+STR_3447    :{BLACK}No mechanics are hired!
```

Comment:
If you add a new string, you must take an ID that's higher than all the other ones. Not an unused ID inbetween.